glue_jupyter/state_traitlets_helpers.py

```diff
@@ -1,12 +1,12 @@
 import json
+import matplotlib
 import traitlets
 from collections import defaultdict
 from traitlets.utils.bunch import Bunch
 from glue.core.state_objects import State
 from glue.core import BaseCartesianData, Subset, ComponentID
 from echo import CallbackList, CallbackDict
 from matplotlib.colors import Colormap
```

Comment:
```suggestion
import traitlets
from collections import defaultdict
from traitlets.utils.bunch import Bunch
from glue.core.state_objects import State
from glue.core import BaseCartesianData, Subset, ComponentID
from glue.utils.matplotlib import MATPLOTLIB_GE_36
from echo import CallbackList, CallbackDict
from matplotlib.colors import Colormap
if MATPLOTLIB_GE_36:
    from matplotlib import colormaps
else:
    from matplotlib import cm as colormaps
```
Thanks for addressing this. At this point there is only a requirement (via glue-core) on matplotlib>=3.2, so the old interface needs to be kept available.